slick.grid.js

```diff
@@ -558,69 +572,59 @@ if (typeof Slick === "undefined") {
         resizeCanvas();
         bindAncestorScrollEvents();
 
-        $container
-            .on("resize.slickgrid", resizeCanvas);
-        $viewport
-            .on("scroll", handleScroll);
+        _container.addEventListener("resize.slickgrid", resizeCanvas);
+        _viewport.forEach(function (view) {
+          view.addEventListener("scroll", handleScroll);
+        });
 
         if (options.enableMouseWheelScrollHandler) {
-          $viewport.each(function (_idx, viewportElm) {
+          _viewport.forEach(function (view) {
             slickMouseWheelInstances.push(Slick.MouseWheel({
-              element: viewportElm,
+              element: view,
               onMouseWheel: handleMouseWheel
             }));
           });
         }
 
-        $headerScroller
-            //.on("scroll", handleHeaderScroll)
-            .on("contextmenu", handleHeaderContextMenu)
-            .on("click", handleHeaderClick)
-            .on("mouseenter", ".slick-header-column", handleHeaderMouseEnter)
-            .on("mouseleave", ".slick-header-column", handleHeaderMouseLeave);
-        $headerRowScroller
-            .on("scroll", handleHeaderRowScroll);
+        _headerScroller.forEach(function (el) {
+            el.addEventListener("contextmenu", handleHeaderContextMenu);
+            el.addEventListener("click", handleHeaderClick);
+        });
 
-        if (options.showHeaderRow) {
-          $headerRow
-            .on("mouseenter", ".slick-headerrow-column", handleHeaderRowMouseEnter)
-            .on("mouseleave", ".slick-headerrow-column", handleHeaderRowMouseLeave);
-        }
+        _headerRowScroller.forEach(function (scroller) {
+           scroller.addEventListener("scroll", handleHeaderRowScroll);
+        });
 
         if (options.createFooterRow) {
-          $footerRow
-            .on("contextmenu", handleFooterContextMenu)
-            .on("click", handleFooterClick);
+          _footerRow.forEach(function (footer) {
+            footer.addEventListener("contextmenu", handleFooterContextMenu)
+            footer.addEventListener("click", handleFooterClick);
+          });
 
-          $footerRowScroller
-              .on("scroll", handleFooterRowScroll);
+          _footerRowScroller.forEach(function (scroller) {
+            scroller.addEventListener("scroll", handleFooterRowScroll);
+          });
         }
 
         if (options.createPreHeaderPanel) {
-          $preHeaderPanelScroller
-              .on("scroll", handlePreHeaderPanelScroll);
-        }
-
-        $focusSink.add($focusSink2)
-          .on("keydown", handleKeyDown);
-        $canvas
-          .on("keydown", handleKeyDown)
-          .on("click", handleClick)
-          .on("dblclick", handleDblClick)
-          .on("contextmenu", handleContextMenu)            
-          .on("mouseenter", ".slick-cell", handleMouseEnter)
```

Comment:
why was this changed from `mouseenter` to `mouseover`? It's not the same event, so why the change?